docs/pages/product/workspace/ai-assistant.mdx

```diff
@@ -97,6 +97,44 @@ to give the AI context on possible values in a categorical dimension:
               - completed
 ```
 
+### Value search
```

Comment:
@mjirv Can we deduplicate these sections and just add a link from the AI Assistant page to the AI API page? I think saying something along the lines of "AI Assistant will use [value search](...) as long as it's enabled for AI API" would suffice.

Reply:
yep, done